ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/inventory/host_vars/SITE1-LER3.yml

```diff
@@ -0,0 +1,4 @@
+# Test when overlay_mpls is false
+evpn_overlay_bgp_rtc: true
+
+fabric_evpn_encapsulation: vxlan
```

Comment:
This should not be tested under the eos_designs-mpls-isis-sr-ldp scenario. This should already be covered plentyful in eos_designs_unit_tests.